_targets.R

```diff
@@ -17,17 +17,15 @@ tar_source(files = "src/mWater_collate")
 tar_source(files = "src/api_pull")
 
 list(
-  # Make sure directory is structured correctly ----
-  # For now we will use the data directory structure that we have been using, but
-  # this will need to be updated with the directory structure in the fc system.
 
-  ## Incoming API data and related API data archive folder ----
+  # Confirm incoming_api_data folder is empty, which indicates the last
+  # iteration of the data pull + QAQC ran properly.
   tar_target(
     name = verify_incoming_data_dir,
     command = {
       check_incoming_api_dir(incoming_dir = "data/api/incoming_api_data/",
                              archive_dir = "data/api/archive_api_data/")
-      },
+    },
```

Comment:
question: If we want this to always happen at the beginning of the pipeline, would that mean that we would `require` itself?